src/Paket/Program.fs

```diff
@@ -130,13 +130,13 @@ try
         else
             Dependencies.Init()
 
-    | Command(InitAutoRestore, args) ->
-        let results = commandArgs<InitAutoRestoreArgs> args
+    | Command(AutoRestore, args) ->
+        let results = commandArgs<AutoRestoreArgs> args
 
-        if results.IsUsageRequested then
```

Comment:
Shouldn't this remain in new version?
I mean if help is requested we shouldn't perform any action even when all parameters for the action are provided.